applicationinsights/requests/WSGIApplication.py

```diff
@@ -49,6 +49,7 @@ def __init__(self, instrumentation_key, wsgi_application, *args, **kwargs):
         self.client = applicationinsights.TelemetryClient(instrumentation_key, telemetry_channel)
         self.client.context.device.type = "PC"
         self._wsgi_application = wsgi_application
+        self._common_properties = kwargs.pop('common_properties', {})
```

Comment:
yes, it will be great if you can add a note into README.md. I see that `DESCRIPTION.rst` is out of sync with readme. @OsvaldoRosado @jjjordanmsft do you know the history here? From looks of it I'd say readme should just point to DESCRIPTION.rst instead of duplicating the context